android/app/src/main/java/labs/pooh/mycanteen/ui/view/RatedFoodView.kt

```diff
@@ -0,0 +1,60 @@
+package labs.pooh.mycanteen.ui.view
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.drawable.Drawable
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.view.isVisible
+import kotlinx.android.synthetic.main.rated_food_view.view.*
+import labs.pooh.mycanteen.R
+
+class RatedFoodView(context: Context) : ConstraintLayout(context) {
+
+    private var numStars: Float = 0F
+        set(value) {
+            assert(value >= 0) { "Rating cannot be negative value" }
```

Comment:
Do assertions run in release builds? If so, I would remove them. Same below.

Reply:
not sure, can be removed in release version in future

Reply:
Let's keep it for now